react-payments-app/src/components/common/Modal/ModalRow.jsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+
+const ModalRow = ({ data, onClick }) => {
+  const [isClicked, setIsClicked] = useState(false);
+  const [selectedId, setSelectedId] = useState(null);
+
+  const handleCompanySelected = (e) => {
+    const selectedId = e.target.parentNode.id;
+    setSelectedId(selectedId);
+    setIsClicked(!isClicked);
+  };
+
+  useEffect(() => {
+    onClick(isClicked, selectedId);
+  }, [isClicked, selectedId]);
+
+  return (
+    <div className='flex-center'>
+      {Array.isArray(data)
+        ? data?.map((company) => (
```

Comment:
```suggestion
        ? data.map((company) => (
```

data가 배열인걸 검증하였기 때문에 옵셔널 체이닝은 없어도 되지 않을까요?

Reply:
[28fae0b](https://github.com/next-step/react-payments/pull/84/commits/28fae0bd67187eaa2c86414820571f22d5db6c05)